R/run_example.R

```diff
@@ -124,10 +124,22 @@ run_example <- function(){
             label_hint("label3", "These are some examples of the types of user
                    feedback inputs that you can use"),
             heading_text("tag_Input", size = "s"),
-            tag_Input("tag1", "NAVY"),
-            tag_Input("tag2", "RED", "red"),
-            tag_Input("tag3", "BLUE", "blue"),
-            tag_Input("tag4", "YELLOW", "yellow"),
+
+            shinyGovstyle::tag_Input("tag1", "Default"),
+            shinyGovstyle::tag_Input("tag2", "Grey", "grey"),
+            shinyGovstyle::tag_Input("tag3", "Green", "green"),
+            shinyGovstyle::tag_Input("tag4", "Turquoise", "turquoise"),
+            shinyGovstyle::tag_Input("tag5", "Blue", "blue"),
+            shinyGovstyle::tag_Input("tag6", "Light-blue", "light-blue"),
+            shinyGovstyle::tag_Input("tag7", "Purple", "purple"),
+            shinyGovstyle::tag_Input("tag8", "Pink", "pink"),
+            shinyGovstyle::tag_Input("tag9", "Red", "red"),
+            shinyGovstyle::tag_Input("tag10", "Orange", "orange"),
+            shinyGovstyle::tag_Input("tag11", "Yellow", "yellow"),
+            shinyGovstyle::tag_Input("tag12", "Incorrect colour", "violet"),
```

Comment:
Is this testing that violet defaults to Default instead? If so think we can remove from this bit and maybe add this into a unit test instead as we'll want to catch any changes to this in future

Reply:
I have removed this from run example but wasn't sure how to include in test that? I had a look at the object in the environment and I'm not sure you can actually pull out the colour (navy)

Reply:
ooo yes good spot you're right it'd need to be a UI shinytest I think, which we've not got in there at the moment. I'll raise as a more general issue but happy if you want to merge 😄